gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -742,16 +750,25 @@ private void updateOnApiChange(
           userHasChangedOutputFile = false; // cancel the modify event from set call.
         }
 
-        boolean showAngleWarning = false;
+        boolean showAngleBar = false;
         if (isAngle(config)) {
           DeviceCaptureInfo dev = devices.get(device.getCombo().getSelectionIndex());
-          if (dev.device.getConfiguration().getAngle().getVersion() == 0 ||
-              dev.device.getConfiguration().getAngle().getVersion() <
-              settings.preferences().getLatestAngleRelease().getVersion()) {
-            showAngleWarning = true;
+          int version = dev.device.getConfiguration().getAngle().getVersion();
+          if (version == 0 || version < settings.preferences().getLatestAngleRelease().getVersion())
+          {
+            if (version == 0) {
+              angleMessage.setText("ANGLE was not found on this device, but is required for selected trace type:");
+              installAngle.setText("Install ANGLE");
```

Comment:
It would be great if we could keep the original color change, maybe yellow for ANGLE update and red for ANGLE install?

Reply:
Yes, Pau had the same concern that the message is less highlighted with the previous color, so we should keep the regular color for now until the theme issues are fixed.

Reply:
Considering that we change to texts to warning colors when other requirements are not fulfilled(Not all field finished; invalid adb; empty app), I am a bit concerned that users will find it inconsistent and think this is not something mandatory(although the continue button will get disabled).  
To make sure the texts are readable, we could 1. mandate foreground color and background color together, and 2. define the color ourselves rather than get the system default color. But I am fine that the theme thing is done in another PR, just that we should remember to do it and not let it get lost.

Reply:
You are right about inconsistent colors in general. In this case I don't see a grave inconsistency, since the angle message bar is a label and button, which is visually very different from the warning messages.

Fixing the theme problem is a bigger issue since SWT inherits OS constraints. That won't get lost.